discovery-provider/plugins/notifications/src/email/notifications/components/Body.tsx

```diff
@@ -46,19 +46,15 @@ const UnreadNotifications = ({ message }) => (
 const snippetMap = {
   ['favorite'](notification) {
     const [user] = notification.users
-    return `${
-      user.name
-    } favorited your ${notification.entity.type.toLowerCase()} ${
-      notification.entity.name
-    }`
+    return `${user.name
```

Comment:
not 100% sure this is the right lint pattern? do you have autolint on your vscode? one thing you can run is `npm run lint:fix` too to double check here

Reply:
ooo i did not thanks for pointing this out! just installednow